subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -703,10 +703,9 @@ void pe_drs_send_swap_run(void *obj)
 		return;
 	}
 
+	/* Transition to PE_SNK_READY on SenderResponseTimer timeout */
```

Comment:
Nit: This commit message should say in what PE state this change is being made.

Reply:
The commit message has been updated